Assets/_PV1/Assets/Geiger System/Scripts/Player.cs

```diff
@@ -0,0 +1,26 @@
+﻿using _PV1.Assets.Geiger_System.Scripts;
+using _PV1.Assets.Geiger_System.Scripts.Interfaces;
+using UnityEngine;
+
+namespace _PV1.Assets.Geiger_System.Scripts
+{
+    public class Player : MonoBehaviour, IDamagable
+    {
+        [SerializeField] private GasMask m_gasMask;
+
+        private float m_healthPoints = 100f;
+        
+        public void TakeDamage(float amount)
+        {
+            Debug.Log($"TakeDMG: {amount}");
+            
+            if (m_gasMask.HasDurability)
+            {
+                m_gasMask.TakeDamage(amount);
+                return;
+            }
+            
+            m_healthPoints -= amount;
+        }
+    }
+}
```

Comment:
```suggestion
﻿using PV.Systems.Geiger
using UnityEngine;

namespace PV.Entities
{
    public class Player : MonoBehaviour, IDamagable
    {
        [SerializeField] private GasMask m_GasMask;

        private float m_HealthPoints = 100f;
        
        public void TakeDamage(float amount)
        {
            Debug.Log($"TakeDMG: {amount}");
            
            if (m_GasMask.HasDurability)
            {
                m_GasMask.TakeDamage(amount);
                return;
            }
            
            m_HealthPoints -= amount;
        }
    }
}
```

Mijn suggestie is dat je de namespace veranderd en let op het gebruik van CamelCasing voor de variabelen.